casper/src/main/resources/RevVault.rho

```diff
@@ -188,11 +188,13 @@ in {
                     for (@result <- authKeyValidCh) {
                       if (result) {
                         new ack in {
-                          logStore!(*logCh) |
-                          // Set logging on the purse so that we also see deposits into the vault
-                          purse!("setLog", *logCh, *ack) |
-                          for (_ <- ack) {
-                            ret!(true)
+                          for (_ <- logStore) {
+                            logStore!(*logCh) |
```

Comment:
Is it better to put this after purse `setLog` ack return?

Reply:
This is setting the log channel for the RevVault immediately and then telling the underlying purse to use the same log channel.  Once that finishes, only then do we return.